rust-install/src/component/package.rs

```diff
@@ -31,6 +32,10 @@ pub trait Package {
                    -> Result<Transaction<'a>>;
 }
 
+pub trait PackageDebug: Package + fmt::Debug {}
+impl<T: Package + fmt::Debug> PackageDebug for T {}
```

Comment:
Instead of introducing `PackageDebug`, can we just have `Package` imply `Debug`.